README.md

```diff
@@ -52,7 +52,8 @@ The `uno` command uses `conf/env.sh.example` for its default configuration which
 sufficient for most users.
 
 Optionally, you can customize this configuration by creating an `env.sh` file and modifying it for
-your environment:
+your environment. Inside this script the variable `UNO_HOME` defaults to the location of the
+cloned Uno repository. You may override this by defining UNO_HOME yourself, if need be.
```

Comment:
I think `You may override this by defining UNO_HOME yourself, if need be.` should be removed.  Before this PR, it was suggested in `load-env.sh` that users can override UNO_HOME.  However, I don't think we should allow this as I don't think it's necessary and adds complexity to scripts.